libkpod/layer/layer.go

```diff
@@ -0,0 +1,12 @@
+package layer
+
+import cstorage "github.com/containers/storage"
+
+// FullID gets the full id of a layer given a partial id or name
+func FullID(store cstorage.Store, id string) (string, error) {
```

Comment:
`store.Layer()` accepts a possibly-truncated ID, but I thought we wanted the full-length ID for some purpose.